meta/SaiInterface.h

```diff
@@ -222,6 +222,22 @@ namespace sairedis
                     _In_ uint32_t attrCount,
                     _In_ const sai_attribute_t *attrList) = 0;
 
+            virtual sai_status_t mdioRegRead(
+                    _In_ sai_object_id_t switch_id,
+                    _In_ uint32_t device_addr,
+                    _In_ uint32_t start_reg_addr,
+                    _In_ uint32_t number_of_registers,
+                    _In_ bool clause_22,
+                    _Out_ uint32_t *reg_val);
+
+            virtual sai_status_t mdioRegWrite(
+                    _In_ sai_object_id_t switch_id,
+                    _In_ uint32_t device_addr,
+                    _In_ uint32_t start_reg_addr,
+                    _In_ uint32_t number_of_registers,
+                    _In_ bool clause_22,
+                    _In_ const uint32_t *reg_val);
+
```

Comment:
if thoes apis are sai_switch_mdio_read_fn and sai_switch_mdio_write_fn, please use correspinding names switchMdioRead and switchMdioWrite for naming, also pleae remove parameter clause_22, sinice originally those apis in saiswitch.h dont have this parameter, and SaiInterface should mimic SAI apis interface

Reply:
there should be 2 sets of api

Reply:
Changed according to your comment.